tools/c7n_gcp/c7n_gcp/actions/notify.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2018 Capital One Services, LLC
```

Comment:
2019 i assume

Reply:
Good catch. Will be fixed in upcoming commit.